rbx_dom_lua/src/EncodedValue.lua

```diff
@@ -205,6 +205,24 @@ types = {
 		end,
 	},
 
+	EnumItem = {
+		fromPod = function(pod)
+			local enum = Enum[pod.type]
+
+			-- If the given EnumItem value is not valid for this Enum, then
+			-- that's pretty weird, but we'll just return the first one in the
+			-- list instead
+			return enum:FromValue(pod.value) or enum:GetEnumItems()[1]
+		end,
```

Comment:
That's the general assumption we make with `Axes` and other potentially fallible types.

In particular it feels like it would be better to get `nil` out of this than it would to get an outright incorrect EnumItem.